demo/movielens-100k-estimator/movielens-100k-estimator.py

```diff
@@ -0,0 +1,169 @@
+  out = tf.reshape(dnn, shape=[-1])
+  loss = tf.keras.losses.MeanSquaredError()(rating, out)
+  predictions = {"out": out}
+
+  if mode == tf.estimator.ModeKeys.EVAL:
+    eval_metric_ops = {}
+    return tf.estimator.EstimatorSpec(mode=mode,
+                                      loss=loss,
+                                      eval_metric_ops=eval_metric_ops)
+
+  if mode == tf.estimator.ModeKeys.TRAIN:
+    optimizer = tf.compat.v1.train.AdamOptimizer(learning_rate=0.001)
+    optimizer = tfra.dynamic_embedding.DynamicEmbeddingOptimizer(optimizer)
+    train_op = optimizer.minimize(
+        loss, global_step=tf.compat.v1.train.get_or_create_global_step())
+    return tf.estimator.EstimatorSpec(mode=mode,
+                                      predictions=predictions,
+                                      loss=loss,
+                                      train_op=train_op)
+
+  if mode == tf.estimator.ModeKeys.PREDICT:
+    predictions_for_net = {"out": out}
+    export_outputs = {
+        "predict_export_outputs":
+            tf.estimator.export.PredictOutput(outputs=predictions_for_net)
+    }
+    return tf.estimator.EstimatorSpec(mode,
+                                      predictions=predictions_for_net,
+                                      export_outputs=export_outputs)
+
+
+def train(model_dir):
+  model_config = tf.estimator.RunConfig(log_step_count_steps=100,
+                                        save_summary_steps=100,
+                                        save_checkpoints_steps=100,
+                                        save_checkpoints_secs=None,
+                                        keep_checkpoint_max=2)
+
+  estimator = tf.estimator.Estimator(model_fn=model_fn,
+                                     model_dir=model_dir,
+                                     params=None,
+                                     config=model_config)
+
+  train_spec = tf.estimator.TrainSpec(input_fn=input_fn)
+
+  eval_spec = tf.estimator.EvalSpec(input_fn=input_fn)
+
+  tf.estimator.train_and_evaluate(estimator, train_spec, eval_spec)
+
+
+def serving_input_receiver_dense_fn():
+  input_spec = {
+      "movie_id": tf.constant([1], tf.int64),
+      "user_id": tf.constant([1], tf.int64),
+      "user_rating": tf.constant([1.0], tf.float32)
+  }
+  return tf.estimator.export.build_raw_serving_input_receiver_fn(input_spec)
+
+
+def export(model_dir, export_dir):
+  model_config = tf.estimator.RunConfig(log_step_count_steps=100,
+                                        save_summary_steps=100,
+                                        save_checkpoints_steps=100,
+                                        save_checkpoints_secs=None)
+
+  estimator = tf.estimator.Estimator(model_fn=model_fn,
+                                     model_dir=model_dir,
+                                     params=None,
+                                     config=model_config)
+
+  estimator.export_saved_model(export_dir, serving_input_receiver_dense_fn())
```

Comment:
Is saving 4 models in line with expectations, when I run `start.sh`?

Reply:
bugfix pr https://github.com/tensorflow/recommenders-addons/pull/70/files